playbooks/roles/systems/defaults/main/images.yml

```diff
@@ -1,3 +1,3 @@
 systems_pgadmin_image: dpage/pgadmin4:6.20
 systems_postgres_image: postgres:12.4
-systems_api_image: tapis/systems:1.5.0
```

Comment:
I was using the dev image here, but we should prrobably switch it back to 1.5.0 ...  not sure why I thought I needed to do this.